client/src/cordova/build.action.mjs

```diff
@@ -34,7 +34,7 @@ export async function main(...parameters) {
   const {platform, buildMode, verbose} = getBuildParameters(parameters);
 
   await runAction('client/src/www/build', ...parameters);
-  await runAction('client/src/tun2socks/build', ...parameters);
```

Comment:
You need to look into the source files inside the framework. Though I think it would fail to build.
Can you at least run the macos or ios client to confirm it still works?